### DISCUSSION TITLE
随机击毙一名绳网网友

### DISCUSSION THREAD

**Discussion by MRScongZZZ** [share121/inter-knot#916] [其他]
![87bdadce8396c56d6886510dc55d687](https://github.com/user-attachments/assets/be73eeb2-64f6-4445-aaff-af070b43f8d8)

**Comment by ZhuYuan-148** [comment]
非法乱用武器，你被逮捕了。

**Comment by themagicmoon** [comment]
给我选最小的子弹😋

**Comment by Hongxuli2006** [comment]
LAPD Drop your motherfuck'in weapon！

**Comment by GanlinSama** [comment]
是治安局的朱鸢警官，我们有救了！

**Comment by Kamisato100** [comment]
但你这不是信号枪么，怎么击毙

**Comment by sdsd122** [comment]
打到我的心了　宝贝

**Comment by Pasukalu** [comment]
补药啊

**Comment by 10kaihua** [comment]
治安局来的就是快，绳网真是越做越好了

**Comment by Tang-Li-1** [comment]
这里里是中国bro 中国不允许开枪！

**Comment by MiaoJiSir** [comment]
先生你已经被治安局盯上了，喵吉劝你好自为之！

**Comment by zifeiyu1111** [comment]
![屏幕截图 2024-08-11 175035](https://github.com/user-attachments/assets/5d040b0b-df93-4f1c-a318-a46470387256)
好好练练枪法吧

**Comment by Larksonguesinaspic** [comment]
github.com/share121/inter-knot/discussions/1280#new_comment_form
看一下
